config/default.yaml

```diff
@@ -2,6 +2,10 @@ server:
   port: 5999
   appRoot: "/bucky"
 
+handlers:
+  http_headers:
+    Access-Control-Allow-Origin: www.app.dev.pardot.com
```

Comment:
This should default to "*".  We'll customize it in a config file as part of deploy.